src/experiments/tagging/6_19_17/IRRGtiff.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I would say that the first two sets of experiment files (from 6//19) had some bugs in them, so they shouldn't be committed.